portfolio/src/main/webapp/comments.js

```diff
@@ -1,12 +1,30 @@
+function addMaxCommentsInputListener(){
+    var maxCommentsInput = document.getElementById("max-comments");
+    maxCommentsInput.addEventListener("change", displayComments);
+}
+
 function displayComments(){
-    fetch('/comments-data')
+    removeAllCommentsFromDOM();
+    var maxComments = getNoMaxComments();
+    console.log(maxComments);
+    fetch('/comments-data?max-comments='+maxComments)
```

Comment:
Is there a way to nicely set the arguments without having to manually to string concatenation?

Reply:
I updated that part using a URL object and the searchParams.append() function